### DISCUSSION TITLE
Can you send SO logs to windows 10 ms word?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#9442] [Unsupported Versions] [answered]
Hi all, I am trying to analyze SO logs and other files. I think it would be easier for me to evaluate them on MS Word or even Word pad for searching. Can I do this?
thanks for any help or advice

**Comment by isaacgolding** [comment, marked as answer]
You would need to use SFTP or SCP to copy the logs out of the system and open them up in windows.  More of a hasle than its worth if you arent familiar with those tools.   You would be better off using something like nano or vi file editor if you wanted to look directly at the log files on the server.     "sudo yum install nano" will get you the nano editor which you can use via ssh or local terminal login to view files directly in an editor like interface.

**Comment by TotieBash** [comment]
The linux tools like cat, more, grep, vi, vim, etc are more powerful tools for for going through linux logs as compared to any Windows gui tools.  Learning these tools are an essential skill to have in your toolbelt and I highly suggest taking the time to do it.  However, if you insist in Windows tools, like the previous comment scp/sftp the file to your Windows. You can use Winscp gui tool to grab the files.  Windows now have scp tool built in the command line, you can grab files through the command line.

**Reply by iqworks** [reply]
thanks totie, i will be looking at that.
